digitaltwins/samples/device/javascript/pnpSampleDevice.js

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+const Protocol = require('azure-iot-device-mqtt').Mqtt;
+const Client = require('azure-iot-device').Client;
+const Message = require('azure-iot-device').Message;
```

Comment:
>; [](start = 51, length = 1)

add:
const { promisify }= require('util');

Reply:
do we need it ? i guess u added it for **promisifying** sleep

Reply:
yup.

---
In reply to: [442561203](https://github.com/Azure/azure-iot-sdk-node/pull/821#discussion_r442561203) [](ancestors = 442561203)